src/Evolution/DiscontinuousGalerkin/SlopeLimiters/MinmodTci.hpp

```diff
@@ -0,0 +1,47 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstdlib>
+#include <utility>
+
+#include "Evolution/DiscontinuousGalerkin/SlopeLimiters/MinmodType.hpp"
+#include "Utilities/Gsl.hpp"
+
+/// \cond
+class DataVector;
+template <size_t Dim, typename T>
+class DirectionMap;
+template <size_t VolumeDim>
+class Element;
+template <size_t VolumeDim>
+class Mesh;
+/// \endcond
+
+namespace Minmod_detail {
+/// \brief Implements the troubled-cell indicator corresponding to the Minmod
```

Comment:
Docs inside a `_detail` namespace won't show up (see point 2 in my comment above)

Reply:
For now I propose to just make this a non-doxygen comment, it will still be useful in that form.

Reply:
I'm fine leaving it as dox comment, since I expect this code, or some wrapper thing to become as part of a public interface.